src/decoderbufs.c

```diff
@@ -595,6 +595,25 @@ static int tuple_to_tuple_msg(Decoderbufs__DatumMessage **tmsg,
     /* set the column name */
     datum_msg.column_name = NameStr(attr->attname);
 
+    /* set is indxed */
```

Comment:
`check if this attr is part of the index`?

Reply:
done